.github/workflows/skywalking.yaml

```diff
@@ -555,26 +555,31 @@ jobs:
           - name: Kafka Log
             config: test/e2e-v2/cases/kafka/log/e2e.yaml
 
-          - name: Istio Metrics Service 1.15.0
+          - name: Istio Metrics Service 1.20.0
             config: test/e2e-v2/cases/istio/metrics/e2e.yaml
             env: |
-              ISTIO_VERSION=1.15.0
-              KUBERNETES_VERSION=25
-          - name: Istio Metrics Service 1.16.0
+              ISTIO_VERSION=1.20.0
+              KUBERNETES_VERSION=28
+          - name: Istio Metrics Service 1.21.0
             config: test/e2e-v2/cases/istio/metrics/e2e.yaml
             env: |
-              ISTIO_VERSION=1.16.0
-              KUBERNETES_VERSION=25
-          - name: Istio Metrics Service 1.17.0
+              ISTIO_VERSION=1.21.0
+              KUBERNETES_VERSION=28
+          - name: Istio Metrics Service 1.22.0
             config: test/e2e-v2/cases/istio/metrics/e2e.yaml
             env: |
-              ISTIO_VERSION=1.17.0
-              KUBERNETES_VERSION=25
-          - name: Istio Metrics Service 1.18.0
+              ISTIO_VERSION=1.22.0
+              KUBERNETES_VERSION=28
+          - name: Istio Metrics Service 1.23.0
             config: test/e2e-v2/cases/istio/metrics/e2e.yaml
             env: |
-              ISTIO_VERSION=1.18.0
-              KUBERNETES_VERSION=25
+              ISTIO_VERSION=1.23.0
+              KUBERNETES_VERSION=28
+          - name: Istio Metrics Service 1.24.0
+            config: test/e2e-v2/cases/istio/metrics/e2e.yaml
+            env: |
+              ISTIO_VERSION=1.24.0
+              KUBERNETES_VERSION=28
 
           - name: Rover with Istio Process 1.15.0
```

Comment:
@mrproliu consider upgrading this istio version to >= 1.20.0 and test whether the eBPF module needs updated, 1.15 is quite old and I don’t think we would like to maintain it anymore